src/controllers/recipesController.js

```diff
@@ -0,0 +1,84 @@
+const BaseController = require('./baseController')
+
+class RecipesController extends BaseController {
+  constructor(a) {
+    super(a)
```

Comment:
Pervadinti `a` į `application`:
```suggestion
  constructor(application) {
    super(application)
```